app/controllers/page_views_controller.rb

```diff
@@ -3,12 +3,23 @@ class PageViewsController < ApplicationMetalController
   # It should help performance.
   include ActionController::Head
 
+  # Each time we have a non-authenticated visitor, we have a 10% chance that we will record that
+  # specific impression as a page view.  When we do record that specific impression as a page view,
+  # we want to give credit for all likely page views that we didn't record (e.g., the 90% or so).
+  #
+  # @note Yes, this is a very verbose constant name.  Apologies, don't type it.  But I [@jeremyf]
+  #       want it here to explain a magic number.
+  #
+  # @see https://github.com/forem/forem/blob/05413f101ab92eb7e25cf5b098919e21179be602/app/assets/javascripts/initializers/initializeBaseTracking.js.erb#L113-L117
```

Comment:
Same with this one, should we point to `main`? (although I guess the line references could fall out of sync if we do, so maybe best to keep as is 🤔 )
```suggestion
  # @see https://github.com/forem/forem/blob/main/app/assets/javascripts/initializers/initializeBaseTracking.js.erb#L113-L117
```

Reply:
Yeah, a head scratcher and part of why I wanted it to be a particular SHA.  "main" makes for a shorter URL but I'm always hesitant to consider how something might move and "get lost".

`main` has the "this is the likely ongoing state" which I believe is most true to intention.